addons/folo/common/src/main/java/org/commonjava/indy/folo/data/DtxBuildRecord.java

```diff
@@ -0,0 +1,59 @@
+package org.commonjava.indy.folo.data;
+
+
+import com.datastax.driver.mapping.annotations.Column;
+import com.datastax.driver.mapping.annotations.PartitionKey;
+import com.datastax.driver.mapping.annotations.Table;
+
+import java.util.Date;
+
+@Table(name = "builds" ,  readConsistency = "QUORUM" , writeConsistency = "QUORUM")
+public class DtxBuildRecord {
```

Comment:
I thought we discussed using just the path records to form the "sealed" record? This data object doesn't seem to add much value, if each path record contains a timestamp. Instead, we could just pull the path records, sort them, and look at timestamps for first and last record to arrive at start/end times for the "sealed" record. If we're not actually compiling the path entries into a record and storing that (which is what we used to do, IIRC), then I'm not sure it's valuable to store this.

Also, for marking a record as sealed, we could have a `sealed` boolean in the path entries db, then execute an `UPDATE...WHERE` statement to flip those at the end of the build.

FWIW, if we want to grab the list of tracking records, we should be able to `SELECT DISTINCT` to get that information from the path entries.

Reply:
This builds table is optional and at this moment is just a placeholder. table if we wanna use some type of starting/finishing timestamp for every buildID/tracking_key.

We have. 'sealed' boolean in. 'records' table.